.github/changelog-configuration.json

```diff
@@ -26,6 +26,7 @@
     "gradle-wrapper"
   ],
   "sort": "ASC",
+  "tag_resolver.filter": "^\\d",
```

Comment:
Sorry I wasn't more clear (their docs aren't clear either) but I think this needs to be a JSON object: `"tag_resolver": { "filter": "^\\d" },`

Reply:
I'm not sure if it needs the `pattern`.

https://github.com/mikepenz/release-changelog-builder-action/commit/ed954db1f67d8c6800eaee966fd43dba85097fac#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L206-L208

Reply:
Oh! Yes! Good catch! It looks like their example is up-to-date:
<img width="439" alt="Screen Shot 2022-02-17 at 8 15 17 AM" src="https://user-images.githubusercontent.com/1420419/154489870-9fe367ac-937d-4e0a-a3ce-eb91419143ad.png">

but their config docs are out of date:
<img width="908" alt="Screen Shot 2022-02-17 at 8 15 08 AM" src="https://user-images.githubusercontent.com/1420419/154489864-49aed832-56e0-4414-a383-2193510d12e7.png">

Anyway, yes, I think that 
```json
"tag_resolver": {
  "filter": {
    "pattern": "^\\d"
  }
}
```
is the way to go.